pkg/controller/infrastructure/actuator.go

```diff
@@ -198,7 +201,21 @@ func (a *actuator) newInitializer(infra *extensionsv1alpha1.Infrastructure, conf
 	return a.terraformerFactory.DefaultInitializer(a.Client(), files.Main, files.Variables, files.TFVars, stateInitializer), nil
 }
 
-func (a *actuator) extractStatus(ctx context.Context, tf terraformer.Terraformer, infraConfig *alicloudv1alpha1.InfrastructureConfig, machineImages []alicloudv1alpha1.MachineImage) (*alicloudv1alpha1.InfrastructureStatus, error) {
+func (a *actuator) convertImageListToV1alpha1(machineImages []apisalicloud.MachineImage) ([]alicloudv1alpha1.MachineImage, error) {
```

Comment:
TBH, it is not that useful. But the image copying action will only be done once for the testing account. This is because only one encrypted VM image will be kept in one account. I would like to enable it in e2e testing configuration, though VM image creating action is not tested normally.